apps/src/sites/studio/pages/levelbuilder.js

```diff
@@ -36,3 +37,4 @@ window.CodeMirror = codemirror;
 // TODO: Remove when global `marked` is no longer required.
 window.marked = marked;
 window.renderer = renderer;
+window.convertXmlToBlockly = convertXmlToBlockly;
```

Comment:
Is this a sign the JS in `_authored_hints.haml` haml below should be extracted to a JS file?

Reply:
yeah, most definitely. I'll add a TODO

Reply:
Or better yet, a Jira task?

Reply:
:+1: https://codedotorg.atlassian.net/secure/RapidBoard.jspa?rapidView=21&projectKey=LP&view=planning&selectedIssue=LP-150